### PR TITLE
TRAN-143: added changes for backend to improve Shipment

### PR DESCRIPTION
**Pull request by sniedzielski**
part of TICKET: https://sd-transit.atlassian.net/browse/TRAN-143

minor changes in "GET Shipment" payload, fix for 'TransporterDetail' filtering

### CODE REVIEW

File: transit/rest_api/forms/shipment/shipment_details.py [hunk 1]
```diff
@@ -22,6 +22,13 @@ def to_representation(self, value):  # noqa: WPS122
         return list(relevant_line_items)
 
 
+class ShipmentOrderMappingOrderNumbersReadonlyField(CharField):
```
Comment (dborowiecki) [inline transit/rest_api/forms/shipment/shipment_details.py:25]: It'll be better if we combine this with `ShipmentOrderMappingCustomerNamesReadonlyField` 
e.g. you can change CharField to Dict field which would return something as: 

```json 
{"orders": [
{"id": 2, "name": "product"}
]}
```

no reason to keep them in separate serializers as they are regarding single object realtion.

**Review by dborowiecki — CHANGES_REQUESTED** [review]
Please make changes to serializes.

File: transit/rest_api/forms/shipment/shipment_details.py (hunk 1 above)
Comment (sniedzielski) [reply transit/rest_api/forms/shipment/shipment_details.py:25]: ok, sure. I create ticket to do this change in the future https://sd-transit.atlassian.net/browse/TRAN-150

**Comment by sniedzielski** [comment]
@dborowiecki I created ticket to make change in serializer https://sd-transit.atlassian.net/browse/TRAN-150. We can improve this later. As I mentioned this change will also affects on frontend side.

**Review by dborowiecki — APPROVED** [review]

**Comment by dborowiecki** [comment]
Thanks, merged.

**Pull request merged**